economytrickle/economytrickle.py

```diff
@@ -367,9 +405,22 @@ def __init__(self, bot):
                     # don't want bot to add to payout
                     if self.bot.user.id in self.activeUsers[sid]:
                         numActive -= 1
-                    trickleAmt = int((numActive - 1) *
-                                     self.settings[sid]["PAYOUT_PER_ACTIVE"] +
-                                     self.tricklePot[sid])
+                    if self.settings[sid]["BASE_PAYOUT_STATE"] == "on":
+                        trickleAmt = int((numActive - 1) *
+                                         self.settings[sid]["PAYOUT_PER_ACTIVE"] +
+                                         self.tricklePot[sid] +
+                                         self.settings[sid]["BASE_PAYOUT"])
+                    elif self.settings[sid]["BASE_PAYOUT_STATE"] == "off":
+                        trickleAmt = int((numActive - 1) *
+                                         self.settings[sid]["PAYOUT_PER_ACTIVE"] +
+                                         self.tricklePot[sid])
```

Comment:
I'd have preferred if you'd removed the `elif` than the `else`
I'd rather have the peace of mind of a catch-all in there.